src/garage/envs/task_onehot_wrapper.py

```diff
@@ -0,0 +1,166 @@
+"""Wrapper for appending one-hot task encodings to individual task envs.
+
+See `~TaskOnehotWrapper.wrap_env_list` for the main way of using this module.
```

Comment:
`MultiTaskOneHotEnv`?

Reply:
Well, each env will only have a single task. I'm trying to keep the name saliently different from `MultiEnvWrapper`. Probably a better question is if I should drop `Wrapper` from the name and move it to `garage.envs.wrappers`.